### ISSUE TITLE
Incorrectly named drivers

### ISSUE THREAD

**Issue by saad-ali** [mapr/mapr-csi#13]
Per the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md#getplugininfo) the name of a CSI driver must conform to the following requirements:

```
The name MUST follow domain name notation format (https://tools.ietf.org/html/rfc1035#section-2.3.1).

It SHOULD include the plugin's host company name and the plugin name, to minimize the possibility of collisions.

It MUST be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), dots (.), and alphanumerics between.

This field is REQUIRED.
```

While CSI 0.x required reverse domain notation,  [CSI 1.0+](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) dropped the "reverse" part of the requirement. The expectation is that the name will be normal (forward) domain notation.

So for CSI 1.0+, instead of `com.mapr.csi-kdf` the driver name should be `kdf.csi.mapr.com` (I would do `.csi.` instead of `csi-`, but that's just personal opinion).

**Comment by sarjeet2013** [comment]
@saad-ali Yes, We continued the old CSI driver (0.3.0) spec name to 1.0.x CSI spec without conforming the new CSI spec for driver names.

If I remember correctly, We reached out in the csi-wg slack channel if the driver name was enforced in reverse vs forward domain notation by CSI plugin or Kubernetes, but the feedback was that it was not strictly enforced but should conform to the CSI spec going forward to be consistent. 

We can definitely look into it. Thanks for reaching out, btw.
